deployment/setup_20.04.sh

```diff
@@ -848,6 +849,68 @@ function webserver() {
   #TODO(@vn): allow installing nginx only without SSL
 }
 
+
+##############################################################################
+# Report cwctl events to hub
+# Globals:
+#   CHATWOOT_HUB_URL
+# Arguments:
+# event_name: Name of the event to report
+# event_data: Data to report
+# installation_identifier: Installation identifier
+# Outputs:
+#   None
+##############################################################################
+function report_event() {
+  local event_name="$1"
+  local event_data="$2"
+
+  CHATWOOT_HUB_URL="https://hub.2.chatwoot.com/events"
+  
+  # Check if the CHATWOOT_HUB_URL is defined
+  if [ -z "$CHATWOOT_HUB_URL" ]; then
+    echo "CHATWOOT_HUB_URL is not defined. Unable to report event."
```

Comment:
Same comment as below. This seems like a debug log, we don't need to show this to the user ,right?

Reply:
Yes, removed.